include/opentracing/span.h

```diff
@@ -27,10 +28,27 @@ class SpanContext {
       const = 0;
 };
 
+struct LogRecord {
+  using Field = std::pair<std::string, Value>;
+
+  SystemTime timestamp;
+  std::vector<Field> fields;
+};
+
 // FinishOptions allows Span.Finish callers to override the finish
 // timestamp.
 struct FinishSpanOptions {
   SteadyTime finish_steady_timestamp;
+
+  // log_records allows the caller to specify the contents of many Log() calls
+  // with a single vector. May be empty.
+  //
+  // None of the LogRecord.timestamp values may be SystemTime() (i.e., they must
+  // be set explicitly). Also, they must be >= the Span's start system timestamp
+  // and <= the finish_steady_timestamp converted to steady timestamp
+  // (or SystemTime::now() if finish_steady_timestamp is default-constructed).
+  // Otherwise the behavior of FinishWithOptions() is undefined.
+  std::vector<LogRecord> log_records;
```

Comment:
Could we change this to unspecified behavior? 

Also,
```
// and <= the finish_steady_timestamp converted to steady timestamp
```
I think you mean `converted to system timestamp` here.

Reply:
Correct, my mistake.